web/src/drafts.ts

```diff
@@ -474,6 +474,26 @@ export function filter_drafts_by_compose_box_and_recipient(
     return _.pick(drafts, narrow_drafts_ids);
 }
 
+export function get_last_draft_based_on_compose_state():
+    | (LocalStorageDraft & {id: string})
```

Comment:
Might be worth giving this type a nice exported name like `Draft`. I guess that type exists as `LocalStorageDraftWithId` today.

Reply:
Ah great, replaced with that